e2e-tests/functions

```diff
@@ -561,20 +564,29 @@ spinup_pxc() {
     apply_config "$config"
 
     desc 'check if all 3 Pods started'
-    wait_for_running "$cluster-proxysql" 1
+    if [ $haproxy = true ]; then
+        wait_for_running "$cluster-haproxy" 3
+    else
+        wait_for_running "$cluster-proxysql" 1
+    fi
```

Comment:
please rebase this changes on brach from PR https://github.com/percona/percona-xtradb-cluster-operator/pull/524